target/linux/sunxi/image/cortexa53.mk

```diff
@@ -40,6 +40,15 @@ define Device/friendlyarm_nanopi-neo2
 endef
 TARGET_DEVICES += friendlyarm_nanopi-neo2
 
+define Device/friendlyarm_nanopi-r1s-h5
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi R1S-H5
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-usb-net-rtl8152 \
```

Comment:
Is this RTC working fine on your board? Does it have a battery backup?

Reply:
no it doesn't have an RTC so I've removed it